package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "build": "webpack",
     "prestart": "webpack",
-    "start": "node ./scripts/start.js"
+    "start": "node ./scripts/start.js",
+    "precommit": "lint-staged"
```

Comment:
I don't have a strong preference, but I try to order the scripts by frequency of use, roughly.